src/Entity/User.php

```diff
@@ -18,7 +18,7 @@
  * @ORM\Entity(repositoryClass=UserRepository::class)
  * @UniqueEntity(fields={"email"}, message="There is already an account with this email")
```

Comment:
@PGBastien Can we use php 8 attributes here also instead of annotations??

Reply:
Hi @najmi9 good idea 👍🏼 
[`7a0b6fa` (#2)](https://github.com/najmi9/mercure-symfony-chat/pull/2/commits/7a0b6faf6f8b75a7ec0ed51c8bd535d387eb7e19)

Reply:
Thanks 🧡.